index.html

```diff
@@ -349,6 +343,42 @@
     </section>
 
     <section>
+      <h2>Extensions to the <a>DocumentOrShadowRoot</a> Mixin</h2>
+
+      <dl title='partial interface DocumentOrShadowRoot' class='idl'>
+         <dt>readonly attribute Element? pointerLockElement</dt>
+         <dd>
+           <dfn title="pointerLockElement"></dfn>
+
+           <p>If the element, which is set as the target for mouse events while the pointer is
+           locked, is a
+           <a href="https://dom.spec.whatwg.org/#concept-shadow-including-descendant">shadow-including descendant</a> of the
+           <a href="http://dom.spec.whatwg.org/#context-object">context object</a>,
+           return the result of the following algorithm:
+           <ol>
+             <li>Let <var>target</var> be the target for mouse events while the pointer is locked.
+             <li>If <var>target</var> is in the same tree as the
+               <a href="http://dom.spec.whatwg.org/#context-object">context object</a>,
+               let <var>adjusted-target</var> be <var>target</var></li>
+             <li>Otherwise: let <var>adjusted-target</var> be the shadow host of <var>target</var> and repeat the following substeps:
```

Comment:
Unfortunately, you can not use "shadow host of A" if A is not DocumentFragment. :(
DOM Standard defines "A's host" only if A is a DocumentFragment, I think.

Instead, you might want to "A's root’s host", for example.